Lib/idlelib/help_about.py

```diff
@@ -63,6 +63,15 @@ def create_widgets(self):
         header = Label(frame_background, text='IDLE', fg=self.fg,
                        bg=self.bg, font=('courier', 24, 'bold'))
         header.grid(row=0, column=0, sticky=W, padx=10, pady=10)
+
+        tk_patchlevel = self.tk.call('info', 'patchlevel')
+        ext = '.png' if tk_patchlevel >= '8.6' else '.gif'
+        iconfn = os.path.join(os.path.abspath(os.path.dirname(__file__)),
```

Comment:
I suspect 'iconfn' abbreviates 'icon file name'.  Too obscure; I'd prefer 'icon' here and below.